ccs/dotnet/dotnet-guestbook/kubernetes-manifests/guestbook-backend.deployment.yaml

```diff
@@ -0,0 +1,33 @@
+﻿# This Deployment manifest defines:
+# - single-replica deployment of the container image, with label "app: dotnet-guestbook-frontend"
+# - Pod exposes port 8080
+# - specify PORT environment variable to the container process
+# Syntax reference https://kubernetes.io/docs/concepts/configuration/overview/
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: dotnet-guestbook-backend
+  labels:
+    app: dotnet-guestbook
+    tier: backend
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: dotnet-guestbook
+        tier: backend
+    spec:
+      containers:
+      - name: backend
+        image: dotnet-guestbook-backend
+        ports:
+        - name: http-server
+          containerPort: 8080
+        - name: debug
+          containerPort: 9229
+        env:
+        - name: PORT
+          value: "8080"

```

Comment:
Can this value be pulled out of the same configmap?

Reply:
see the other comment on config map